src/components/nodes/task/task.vue

```diff
@@ -11,6 +11,7 @@
     :process-node="processNode"
     :plane-elements="planeElements"
     :is-rendering="isRendering"
+    :boundary-event-dropdown-data="boundaryEventDropdownData"
     @remove-node="$emit('remove-node', $event)"
     @add-node="$emit('add-node', $event)"
     @save-state="$emit('save-state', $event)"
```

Comment:
Something we've been doing is replacing all the individual event listeners to a single `v-on="$listeners"` binding. This may help clean up the component since these event are simply being emitted back up as-is.